src/pretix/eventyay_common/utils.py

```diff
@@ -0,0 +1,64 @@
+import hashlib
+import logging
+import random
+import string
+from datetime import datetime, timedelta, timezone
+
+import jwt
+from django.conf import settings
+
+
+logger = logging.getLogger(__name__)
+
+
+def generate_token(request):
+    """
+    Generate token for video system
+    @param request: user request
+    @return: jwt
+    """
+    uid_token = encode_email(request.user.email)
```

Comment:
The `uid_token` seems not to carry any info for client side to extract. If so, please use a short method with `secrets` module to generate it, as @sourcery-ai suggested.

Reply:
Why do you still have to create `uid_token` in a very complex way?